Backend/Interview.Domain/Rooms/Service/RoomService.cs

```diff
@@ -164,6 +165,59 @@ public async Task<IPagedList<RoomPageDetail>> FindPageAsync(
         });
     }
 
+    public async Task<List<RoomCalendarItem>> GetCalendarAsync(RoomCalendarRequest filter, CancellationToken cancellationToken = default)
+    {
+        var currentUserId = _currentUserAccessor.UserId;
```

Comment:
It might be worth adding a start and end date restriction:
1. The date cannot be more than one month
2. The end date cannot be earlier than the start date

What do you think?